clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -9322,6 +9322,113 @@ static void handleOpenCLAccessAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
   D->addAttr(::new (S.Context) OpenCLAccessAttr(S.Context, AL));
 }
 
+static constexpr std::pair<Decl::Kind, StringRef>
+MakeDeclContextDesc(Decl::Kind K, StringRef SR) {
+  return std::pair<Decl::Kind, StringRef>{K, SR};
+}
+
+// FIXME: Can we perform a lookup instead?
```

Comment:
What do you mean here, exactly?

Reply:
I was wondering if there was a better way to do this (like `LookUpResult ` and compare or something) but it doesn't make sense to do that here. So I will be keeping this code and removing this comment.